std/StringTools.hx

```diff
@@ -413,6 +413,15 @@ class StringTools {
 		return false;
 		#end
 	}
+	
+	/**
+		Converts the first character of the String `s` to upper case and the 
+		remaining characters to lower case.
```

Comment:
Hm.. I wouldn't expect a `capitalize()` function to affect any characters but the first.